story-generator/src/app/app.css

```
@@ -18,7 +18,10 @@
   --accent-2: #8e4b21;
   --accent-3: #6f1d33;
   --accent-ink: #fff9ee;
+  --card-bg: color-mix(in srgb, var(--paper-strong) 86%, var(--paper) 14%);
+  --focus-line: color-mix(in srgb, var(--accent) 62%, var(--paper) 38%);
   --hero-image: url("/story-lab-atmosphere.png");
+  --serif: Georgia, "Times New Roman", serif;
```

Stylelint (17.13.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)